Podcast/FollowButton.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  FollowButton.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 11/9/16.
+//  Copyright © 2016 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class FollowButton: UIButton {
+    
+    init() {
+        super.init(frame: .zero)
+        isSelected = false
+        titleLabel?.font = UIFont.systemFont(ofSize: 14, weight: UIFontWeightSemibold)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func draw(_ rect: CGRect) {
+        let ctx:CGContext = UIGraphicsGetCurrentContext()!;
+        let strokeColor: CGColor = self.tintColor.cgColor
```

Comment:
no need for `self` here and in this file in general since without it, it would default to `self`